src/test/java/uk/gov/hmcts/sptribs/dmn/CamundaTaskWaInitiationTest.java

```diff
@@ -0,0 +1,885 @@
+                List.of(
+                    Map.of(
+                        "taskId", REVIEW_LIST_CASE_LO_TASK,
+                        "name", "Review List Case - Legal Officer",
+                        "workingDaysAllowed", 1,
+                        "processCategories", PROCESS_CATEGORY_HEARING,
+                        "workType", DECISION_WORK_TYPE,
+                        "roleCategory", ROLE_CATEGORY_LO
+                    )
+                )
+            ),
+            Arguments.of(
+                "refer-to-legal-officer",
+                "AwaitingHearing",
+                Map.of("Data", Map.of("cicCaseReferralTypeForWA", "Listed case (within 5 days)")),
+                List.of(
+                    Map.of(
+                        "taskId", REVIEW_LIST_CASE_WITHIN_5DAYS_LO_TASK,
+                        "name", "Review list case (within 5 days) - Legal Officer",
+                        "workingDaysAllowed", 1,
+                        "processCategories", PROCESS_CATEGORY_HEARING,
+                        "workType", DECISION_WORK_TYPE,
+                        "roleCategory", ROLE_CATEGORY_LO
+                    )
+                )
+            ),
+            Arguments.of(
+                "refer-to-legal-officer",
+                "AwaitingHearing",
+                Map.of("Data", Map.of("cicCaseReferralTypeForWA", "Postponement request")),
+                List.of(
+                    Map.of(
+                        "taskId", REVIEW_POSTPONEMENT_REQ_LO_TASK,
+                        "name", "Review Postponement request - Legal Officer",
+                        "workingDaysAllowed", 1,
+                        "processCategories", PROCESS_CATEGORY_HEARING,
+                        "workType", DECISION_WORK_TYPE,
+                        "roleCategory", ROLE_CATEGORY_LO
+                    )
+                )
+            ),
+            Arguments.of(
+                "refer-to-legal-officer",
+                "CaseClosed",
+                Map.of("Data", Map.of("cicCaseReferralTypeForWA", "Reinstatement request")),
+                List.of(
+                    Map.of(
+                        "name", "Review Reinstatement request - Legal Officer",
+                        "workType", DECISION_WORK_TYPE,
+                        "taskId", REVIEW_REINSTATEMENT_REQ_LO_TASK,
+                        "workingDaysAllowed", 5,
+                        "processCategories", PROCESS_CATEGORY_APPLICATION,
+                        "roleCategory", ROLE_CATEGORY_LO
+                    )
+                )
+            ),
+            Arguments.of(
+                "refer-to-legal-officer",
+                "*",
+                Map.of("Data", Map.of("cicCaseReferralTypeForWA", "Other")),
+                List.of(
+                    Map.of(
+                        "taskId", REVIEW_OTHER_REQ_LO_TASK,
+                        "name", "Review other request - Legal Officer",
```

Comment:
@oleon23 - is this not the line in question? Appears to be same as the commit by @gmmagruder